packages/blade/src/components/Button/BaseButton/__tests__/BaseButton.web.test.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.web';
+import BaseButton from '../BaseButton';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+describe('<BaseButton />', () => {
+  it('should render button with default properties', () => {
+    const buttonText = 'Pay Now';
+    const { container, getByText, getByRole } = renderWithTheme(
+      <BaseButton>{buttonText}</BaseButton>,
+    );
+    expect(container).toMatchSnapshot();
+    expect(getByRole('button')).toBeInTheDocument();
```

Comment:
❤️

Reply:
we don't have something similar to test this on React Native too?

Reply:
There is a `getByRole` for react native but we have to explicitly pass `accessibilityRole='button'` to our component. Held this off since we weren't doing a11y in this PR. We could probably add a test for it once we add some a11y props to the button